src/templates/services/index.hbs

```diff
@@ -80,25 +80,25 @@
     </section>
 
     <section id="illustration">
-      <div class="parallax layer-1">
+      <div data="1" class="parallax layer-1">
         <div>
           <img src="/assets/img/services/sad-girl.svg">
           <img src="/assets/img/services/drink.svg">
         </div>
       </div>
-      <div class="parallax layer-2">
+      <div data="2" class="parallax layer-2">
         <div>
           <img src="/assets/img/services/2girls.svg">
           <img src="/assets/img/services/girl.svg">
         </div>
       </div>
-      <div class="parallax layer-3">
+      <div data="3" class="parallax layer-3">
         <div>
           <img src="/assets/img/services/kangaroo.svg">
           <img src="/assets/img/services/donut.svg">
         </div>
       </div>
-      <div class="parallax layer-4">
+      <div data="4" class="parallax layer-4">
```

Comment:
You can indeed create your own attributes. Convention is to start them with `data-...` so they don't interfere with native attributes and are clearly made up by hoomans. So you can add `data-richard` to your `<div>` and then get or set that info with jQuery:
```
$('div#something').attr('data-richard');
$('div#something').attr('data-richard', newValue);
```
Perhaps also read this one:
https://api.jquery.com/data/